osu.Game/Skinning/Components/BeatmapAttributeText.cs

```diff
@@ -225,6 +228,9 @@ private LocalisableString getValueString(BeatmapAttribute attribute)
                 case BeatmapAttribute.StarRating:
                     return (starDifficulty?.Stars ?? 0).ToLocalisableString(@"F2");
 
+                case BeatmapAttribute.MaxPP:
+                    return (starDifficulty?.PerformanceAttributes?.Total ?? 0).ToLocalisableString(@"F2");
```

Comment:
I find it a bit weird how this is rounded to 2 digits when the rolling pp counter that you can add to HUD rounds to full points. This leads to funky situations wherein you seem to get "more pp than the maximum" which actually is just a rounding artifact:

![osu_2024-11-01_13-28-05](https://github.com/user-attachments/assets/2247b1ec-bbea-4350-899c-d45eda015d5e)

This is also the only display in the entire game that displays fractional performance points as far as I can tell. HUD counter doesn't, results screen display doesn't, performance breakdown doesn't.

Reply:
That's a good point. I've rounded it.